docs/source/dataset_summary.rst

```diff
@@ -41,6 +41,7 @@ Motor Imagery
         :class:`Shin2017B`,29,30,2,30,10s,200Hz,3,1,5220
         :class:`Weibo2014`,10,60,7,80,4s,200Hz,1,1,5600
         :class:`Zhou2016`,4,14,3,160,5s,250Hz,3,2,11496
+        :class:`Stieger2021`,62,64,4,?,3s,1000Hz,10,1,250000
```

Comment:
How many trials do we have for the first subject, @rkobler?

Reply:
I should have some time over the weekend to have a look into your questions.

Reply:
Because of the experimental paradigm, the length of each trial actually varies [1]. However, in each session a subject was doing a total of 450 trials. So, I think we should state 450 here. 

Also, I noticed that the figshare respository [2] only contains complete records for the first 47 participants. I plan to reach out to Prof. He to inquire about the missing records. For now, we should change the number of participants to 47. Also, the number of sessions differ across participants. They are either 7 or 11 [1].

[1] https://www.nature.com/articles/s41597-021-00883-1#Sec6
[2]

Reply:
Never mind the previous comment about missing paticipants. It seems like the web api of figshare has an issue. When, I use the API via moabb, I get records for all the 62 subjects.